modules/rhcos-install-iscsi-ibft.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rhcos-install-iscsi-ibft_{context}"]
+= Installing {op-system} on an iSCSI boot device using iBFT
+
+On a completely diskless machine, the iSCSI target and initiator values can be passed through iBFT. iSCSI multipathing is also supported.
+
+.Prerequisites
+. You are in the {op-system} live environment.
+. You have an iSCSI target you want to install {op-system} on.
+. Optional: you have multipathed your iSCSI target.
+
+.Procedure
+
+. Mount the iSCSI target from the live environment:
++
+[source,text]
+----
+$ iscsiadm \
+    --mode discovery \
+    --type sendtargets
+    --portal 192.168.0.10 \ <1>
+    --login
+----
+<1> The IP address of the target portal.
+
+. Optional: enable multipathing and start the daemon:
++
+[source,text]
+----
+$ mpathconf --enable && systemctl start multipathd.service
+----
+
+. Install {op-system} onto the iSCSI target using the necessary kernel arguments, specifying `rd.iscsi.firmware=1`:
++
+[source,text]
+----
+$ coreos-installer install \
+    /dev/disk/by-path/<IP_address>:<port>-iscsi-<target_iqn>-lun-<lun> \
```

Comment:
Sorry, I should've mentioned this in my previous review, but this should be the multipathed disk instead (e.g. `/dev/mapper/mpatha`) if multipath is enabled. Seems fine to me to tweak in a follow-up!